Lib/re/__init__.py

```diff
@@ -282,24 +283,9 @@ def _compile(pattern, flags):
     return p
 
 @functools.lru_cache(_MAXCACHE)
-def _compile_repl(repl, pattern):
+def _compile_template(pattern, repl):
```

Comment:
`_compile_repl` is a clearer name, or add a comment `# compile repl`

Reply:
I intentionally changed the order of parameters and renamed the function because:

1. The returning value is changed. Changing the name helps to avoid some bugs,
2. It is a preparation for adding a public API for compiling replacement strings.

The comment on the following line already explains the purpose of this function, so no new comment is needed.